pkg/systemlogmonitor/logwatchers/filelog/log_watcher_test.go

```diff
@@ -182,7 +183,12 @@ Jan  2 03:04:05 kernel: [2.000000] 3
 }
 
 func TestGoroutineLeak(t *testing.T) {
-	orignal := runtime.NumGoroutine()
+	original_prof := pprof.Lookup("goroutine")
```

Comment:
Should those changes to `TestGoroutineLeak` be moved to the other PR?

Reply:
Done.